tailwind.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     './src/pages/**/*.{js,ts,jsx,tsx}',
     './src/styles/**/*.{js,ts,jsx,tsx}',
   ],
-  darkMode: 'class',
+  darkMode: 'selector',
```

Comment:
I can see that class was replaced with selector from tailwind 3.4.1 but do we see any bugs / deviations? We were using `class` option in darkMode for these many days?